ledger/tracker_test.go

```diff
@@ -142,61 +143,111 @@ func TestTrackerScheduleCommit(t *testing.T) {
 	a.Equal(expectedOffset, dc.offset)
 }
 
-type producePrepareBlockingTracker struct {
-	produceReleaseLock       chan struct{}
-	prepareCommitEntryLock   chan struct{}
-	prepareCommitReleaseLock chan struct{}
-	cancelTasks              bool
+type emptyTracker struct{}
```

Comment:
added emptyTracker as a base type for producePrepareBlockingTracker and errTracker